GVRf/Framework/framework/src/main/java/org/gearvrf/GVRConfigurationManager.java

```diff
@@ -142,6 +142,16 @@ void setDockListenerRequired(boolean value){
      */
     public abstract boolean isHmtConnected();
 
+    /**
+     * @return true if using OVR_Multiview extension, false otherwise
+     */
+    private static native boolean nativeUsingMultiview(long ptr);
+
+    public final boolean usingMultiview()
```

Comment:
Please remove the final keyword. The Daydream backend doesn't compile otherwise.